waku/protocol/v2/waku_filter.nim

```diff
@@ -4,18 +4,18 @@ import
   libp2p/protocols/pubsub/pubsubpeer,
   libp2p/protocols/pubsub/floodsub,
   libp2p/protocols/pubsub/gossipsub,
-  libp2p/protocols/pubsub/rpc/[messages, protobuf],
   libp2p/protocols/protocol,
   libp2p/protobuf/minprotobuf,
   libp2p/stream/connection,
-  ./message_notifier
+  ./message_notifier,
+  ./../../node/v2/waku_types
 
 # NOTE This is just a start, the design of this protocol isn't done yet. It
 # should be direct payload exchange (a la req-resp), not be coupled with the
 # relay protocol.
 
 const
-  WakuFilterCodec* = "/vac/waku/filter/2.0.0-alpha4"
+  WakuFilterCodec* = "/vac/waku/filter/2.0.0-alpha5"
```

Comment:
https://github.com/vacp2p/specs/pull/184#issuecomment-693139372 you bumped all versions, so now we have to do the same everywhere. Next time only bump version if there are actually changes to that protocol.